libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -497,6 +497,33 @@ class BreezSDKMapper {
             "lastRedeemError": swapInfo.lastRedeemError
         ]
     }
+
+    static func dictionaryOf(reverseSwapPairInfo: ReverseSwapPairInfo) -> [String: Any?] {
```

Comment:
Can be `[String: Any]` as there are no optional values

Reply:
Done